server.conf.in

```diff
@@ -109,6 +109,7 @@ serverRealm = local
 ; data: URL for images.
 ; The currently recommended CSP is:
 ;   default-src 'self';
+;   frame-src 'self' data:;
```

Comment:
I am not sure if this is really a good thing. This means essentially that xss could load arbitrary HTML code in an iframe. Please check.